services/auth/session.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 The Gitea Authors. All rights reserved.
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
thanks for this PR, no need to update the date of copyright, we keep it as the date of when the file was first created, and then we rely on git blame for when the file is updated.

Reply:
Fixed.